src/m365/spo/commands/web/web-set.ts

```diff
@@ -182,8 +187,23 @@ class SpoWebSetCommand extends SpoCommand {
       const searchScope = args.options.searchScope.toLowerCase();
       payload.SearchScope = SpoWebSetCommand.searchScopeOptions.indexOf(searchScope);
     }
+    if (typeof args.options.welcomePage !== 'undefined') {
```

Comment:
Let's make sure that this request is within the `try` block so we can catch the potential error.